repokid/__init__.py

```diff
@@ -60,8 +75,10 @@ def init_logging():
     Returns:
         None
     """
+    logging_format = "%(asctime)s - %(levelname)s - %(name)s - [%(filename)s:%(lineno)s - %(funcName)s() ] - %(message)s"  # noqa: E501
+
     if CONFIG:
-        logging.config.dictConfig(CONFIG["logging"])
+        logging_format = CONFIG["logging"]
```

Comment:
does this work?

Reply:
Yes, we later do a `logging.basicConfig(level="DEBUG", format=logging_format)`. With the previous version, logs are duplicated.